### PR TITLE
Add whitespace between Back and Edit links

### LINKED ISSUE #18: Missing whitespace between Edit and Back links when viewing a project
image

### PR DESCRIPTION
**Pull request by Serneum**
Closes #18

### CODE REVIEW

**Comment by Serneum** [comment]
Note: `&nbsp;` ends up adding two spaces for some reason. Adding whitespace at the end of the first link is ignored. Best solution will be one of the following:
1. Put both links on the same link in code
2. Add CSS to add padding/marging

File: web/templates/user/show.html.eex [hunk 1]
```diff
@@ -19,5 +19,5 @@
 
 </ul>
 
-<%= link "Edit", to: user_path(@conn, :edit, @user) %>
+<%= link "Edit", to: user_path(@conn, :edit, @user) %>&nbsp;
```
Comment (ericworkman) [inline web/templates/user/show.html.eex]: You could

```
<%= link "Edit", to: user_path(@conn, :edit, @user), class: "btn btn-default btn-xs" %>
```

Comment (Serneum) [reply web/templates/user/show.html.eex]: Do we want links that look like buttons instead of links?

Comment (ericworkman) [reply web/templates/user/show.html.eex]: It was mostly an example of how to add class to the buttons. I don't care either way, but the buttons in bootstrap deal with spacing pretty well on their own.

Comment (mbramson) [reply web/templates/user/show.html.eex]: +1 for solving this with css rather than throwing a &nbsp in there.

**Comment by Serneum** [comment]
This is ready for review

**Review by mbramson — APPROVED** [review]
LGTM!

**Pull request merged**